src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -145,41 +136,41 @@ private void assertCommandFailure(String inputCommand, Class<? extends Throwable
-            @Override
-            public void saveEduTrack(ReadOnlyEduTrack addressBook, Path filePath)
-                    throws IOException {
-                throw e;
-            }
-        };
-
-        JsonUserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(
-                temporaryFolder.resolve("ExceptionUserPrefs.json"));
-        StorageManager storage = new StorageManager(addressBookStorage, userPrefsStorage);
-
-        logic = new LogicManager(model, storage);
-
-        // Triggers the saveAddressBook method by executing an add command
-        String addCommand = AddCommand.COMMAND_WORD + NAME_DESC_AMY + PHONE_DESC_AMY
-                + EMAIL_DESC_AMY + ADDRESS_DESC_AMY;
-        Student expectedPerson = new StudentBuilder(AMY).build();
-        ModelManager expectedModel = new ModelManager();
-        expectedModel.addStudent(expectedPerson);
-        assertCommandFailure(addCommand, CommandException.class, expectedMessage, expectedModel);
-    }
+    //    /**
+    //     * Tests the Logic component's handling of an {@code IOException} thrown by the
+    //     * Storage component.
+    //     *
+    //     * @param e               the exception to be thrown by the Storage component
+    //     * @param expectedMessage the message expected inside exception thrown by the
+    //     *                        Logic component
+    //     */
+    //    private void assertCommandFailureForExceptionFromStorage(IOException e, String expectedMessage) {
+    //        Path prefPath = temporaryFolder.resolve("ExceptionUserPrefs.json");
+    //
+    //
+    //        // Inject LogicManager with an AddressBookStorage that throws the IOException e when saving
+    //        JsonEduTrackStorage addressBookStorage = new JsonEduTrackStorage(prefPath) {
+    //            @Override
+    //            public void saveEduTrack(ReadOnlyEduTrack addressBook, Path filePath)
+    //                    throws IOException {
+    //                throw e;
+    //            }
+    //        };
+    //
+    //        JsonUserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(
+    //                temporaryFolder.resolve("ExceptionUserPrefs.json"));
+    //        StorageManager storage = new StorageManager(addressBookStorage, userPrefsStorage);
+    //
+    //        logic = new LogicManager(model, storage);
+    //
+    //        // Triggers the saveAddressBook method by executing an add command
+    //        String addCommand = AddCommand.COMMAND_WORD + NAME_DESC_AMY + PHONE_DESC_AMY
+    //                + EMAIL_DESC_AMY + ADDRESS_DESC_AMY;
+    //        Student expectedPerson = new StudentBuilder(AMY).build();
+    //        ModelManager expectedModel = new ModelManager();
+    //        expectedModel.addStudent(expectedPerson);
+    //        assertCommandFailure(addCommand, CommandException.class, expectedMessage, expectedModel);
+    //    }
```

Comment:
Is this part of the test not going to be used? If so maybe can remove

Reply:
Thanks for pointing it out! I've removed this test